### PR TITLE
add main menu gui page

### PR DESCRIPTION
**Pull request by Omarzintan**
This PR implements a main menu where the user can choose their preferred wake phrase and feature list to implement. Like so:

Screenshot 2021-05-18 at 6 44 35 PM
Screenshot 2021-05-18 at 6 44 43 PM
Screenshot 2021-05-18 at 6 44 51 PM

PROBLEM: Just tested other features in Bumblebee and it seems the implementation of PySimpleGUI messes with other implementations of Tkinter in other features, where the tkinter windows just freeze after being used and everything hangs.
POSSIBLE SOLUTION: Changing all other windows to PySimpleGUI seems to do the trick. Im proceeding to change all the tkinter windows to PySimpleGUI.

### CODE REVIEW

File: features/add_contact.py [hunk 1]
```diff
@@ -15,66 +15,34 @@ def __init__(self, bumblebee_api):
 
     def action(self, spoken_text):
         try:
-            self.add_contact_details()
+            self.add_contact_details2()
```
Comment (RexfordEssilfie) [inline features/add_contact.py:18]: Is this defined?

File: main.py [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+
+    # Create the Window and read inputs
+    event, values = sg.Window("Bumblebee Main Menu", layout).read(close=True)
+    wake_phrase = values['wake_phrase']
+    feature_list_name = values['feature_list_name']
+    config_file = values['config_file']
+
+    bee_runner = Bumblebee(
+        name=wake_phrase, feature_list_name=feature_list_name,
+        config_yaml_name=config_file)
```
Comment (RexfordEssilfie) [inline main.py:34]: Nice!

**Review by RexfordEssilfie — APPROVED** [review]
I like the changes here, @Omarzintan! `PySimpleGUI` seems cleaner and easier to use 👌 

Question:
- Does the user have to configure the wake word detector every time? We can add an option to save my change/don't ask again, and then have a feature to allow the user to spin up the settings window again to make changes?

File: features/add_contact.py (hunk 1 above)
Comment (Omarzintan) [reply features/add_contact.py:18]: this should have been changed back to `add_contact_details()`

File: main.py (hunk 2 above)
Comment (Omarzintan) [reply main.py:34]: IKR!

**Comment by Omarzintan** [comment]
> I like the changes here, @Omarzintan! `PySimpleGUI` seems cleaner and easier to use 👌
> 
> Question:
> 
> * Does the user have to configure the wake word detector every time? We can add an option to save my change/don't ask again, and then have a feature to allow the user to spin up the settings window again to make changes?

Yeah I thought about adding that eventually. I'm not sure yet how the user would set it so that it will pop-up again in case they want to change the wake word detector. Trying to think of a way that could be possible without having them go into the code again.

**Pull request merged**